src/lib/gssapi/krb5/k5sealv3.c

```diff
@@ -312,6 +312,8 @@ gss_krb5int_unseal_token_v3(krb5_context *contextptr,
     krb5_key key;
     krb5_cksumtype cksumtype;
 
+    memset(&plain, 0, sizeof(krb5_data));
```

Comment:
We can initialize plain to empty_data() where it is declared, which sets the magic field to the nominally correct value for a krb5_data object.